expression/schema.go

```diff
@@ -233,3 +234,25 @@ func MergeSchema(lSchema, rSchema *Schema) *Schema {
 func NewSchema(cols ...*Column) *Schema {
 	return &Schema{Columns: cols}
 }
+
+// NamingForMySQLProtocol records the names used for mysql protocol.
+type NamingForMySQLProtocol struct {
```

Comment:
1. How about `s/NamingForMySQLProtocol/FieldName/`?
2. Can we put this struct to package `types`, together with `FieldType`?

Reply:
Sure